plugins/api/src/switch_user.cpp

```diff
@@ -73,6 +79,27 @@ namespace
         return 0;
     } // check_input
 
+    auto update_server_to_server_connections(RsComm& _comm) -> void
+    {
+        const auto update_zone_connection = [&client = _comm.clientUser](rodsServerHost& _host)
+        {
+            if (_host.conn) {
+                const auto ec = rc_switch_user(_host.conn, client.userName, client.rodsZone);
+                log_api::info("rc_switch_user ec = [{}]", ec);
```

Comment:
debug?

Reply:
This no longer applies due to code changes.